server/src/main/java/org/cloudfoundry/identity/uaa/cypto/EncryptionKeyService.java

```diff
@@ -18,13 +17,16 @@ public class EncryptionKeyService {
     public EncryptionKeyService(
             final @Value("${encryption.active_key_label}") String activeKeyLabel,
             final @Value("#{@config['encryption']['encryption_keys']}") List<EncryptionKey> encryptionKeys) {
-        if (Strings.isEmpty(activeKeyLabel)) {
+        if (activeKeyLabel == null || activeKeyLabel.isEmpty()) {
             throw new NoActiveEncryptionKeyProvided(
               "UAA cannot be started without encryption key value uaa.encryption.active_key_label"
             );
         }
 
-        List<EncryptionKey> keysWithoutPassphrase = encryptionKeys.stream().filter(encryptionKey -> Strings.isEmpty(encryptionKey.getPassphrase())).collect(Collectors.toList());
+        List<EncryptionKey> keysWithoutPassphrase = encryptionKeys.stream().filter(encryptionKey -> {
+            final String passphrase = encryptionKey.getPassphrase();
+            return passphrase == null || passphrase.isEmpty();
+        }).collect(Collectors.toList());
```

Comment:
This change makes the code worse IMO. The null check was nicely hidden in the util method, and now it pops out.